rules.py

```diff
@@ -953,25 +953,30 @@ def check(self, element):
         object_id = element.get("objectId")
         if element.tag == "Candidate":
             ballot_name = element.find("BallotName")
-            if ballot_name.find("Text") is not None:
```

Comment:
No need to check here as BallotName is required.

Reply:
Updated as per requirement. Added few checks for corner cases as well.